### DISCUSSION TITLE
How can I save the best model during the rtmo model training?

### DISCUSSION THREAD

**Discussion by nepulepu** [open-mmlab/mmpose#2962] [Q&A] [answered]
The default config is using CocoMetric as the evaluator, but during the on the checkpoint hook what is the "save best" metric that can I use to choose the best weight of the model as it stated 

 key_score = metrics[key_indicator]
KeyError: 'CocoMetric

if savebest='CocoMetric'

I am using config from this file:
[RTMO-m](https://github.com/open-mmlab/mmpose/blob/main/configs/body_2d_keypoint/rtmo/coco/rtmo-m_16xb16-600e_coco-640x640.py)

**Comment by Tau-J** [comment, marked as answer]
hi @nepulepu , you can refer to [this doc](https://mmpose.readthedocs.io/en/dev-1.x/user_guides/configs.html#general)
